docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
       - ./fec:/code/fec
       - ./htmlcov:/code/htmlcov
     environment:
-      DB_HOST: db
+      DB_HOSTNAME: db
```

Comment:
this was just wrong, not sure how long that's been there

Reply:
oops!